src/main/java/seedu/duke/question/QuestionInterface.java

```diff
@@ -0,0 +1,31 @@
+package seedu.duke.question;
+
+import seedu.duke.HintInterface;
+import seedu.duke.OptionInterface;
+import seedu.duke.OptionListInterface;
+
+import java.util.ArrayList;
+
+public interface QuestionInterface {
+
+    // might want to change all the interface classes to just the classes themselves
```

Comment:
Hey, I think we might as well just keep the interfaces for easy reference to the behaviors

Reply:
Hey! Thanks for the quick response! Able to make timely adjustments because of that. Have made a new commit with the necessary changes! Do check it out if you are free!